README.md

```diff
@@ -108,6 +108,13 @@ Optional String: Allows you to override the command that is run to gather metric
 
 ### Grepping Metrics
 
+The metrics come in a JSON hash on one line. In order to convert the metric files into a pretty format, they can be processed with `python -m json.tool` like below.
```

Comment:
Why the change to one-liners?  It was nice to be able to read the files without extra processing.

Reply:
The oneliner changes are to accommodate a change to not pretty-printing the JSON files. We are trying to minimize the amount of space that these metrics take on disk, but more importantly when being sent up to support. We can reduce our collection by hundreds of megabytes using this method as well as uncompressed processing size.

@Sharpie did an analysis of the space savings. 

> I did some benchmarking and it turns out that dropping the extra whitespace created by pretty_generate is going to save us significant amounts of space.
>
> For a monolithic master with 8 compilers and one deployed Code Manager environment, the daily archive size was 16 MB. Removing the file-sync-storage-service and pe- datasets drops that to 13 MB.
>
> Removing the whitespace created by pretty_generate and keeping the data drops the archive size to 7.8 MB.
>
> Removing both the whitespace and the data drops it to 5.6 MB. Disk space required for uncompressed files was reduced from 200 MB to 50 MB once everything was removed.
>
> I would expect the savings from removing file-sync-storage-service to increase rapidly as the number of environments grows, but killing off the whitespace is still a significant reduction in size.

Reply:
ok, thanks for the data. :)   Wish there was an easy way to convert on the command line for ruby... since we know that is already installed.

Reply:
Agreed. Would it be better off if I included something in `bin/puppet-metrics-collector` to do this conversion instead of the one-liner? Then the user could run something like `./bin/puppet-metrics-collector pretty-print`.

Reply:
That is a great idea.  But while I would like it.  The real question is do the customers actually care.  Cause if they don't ever look at the raw files then it isn't worth adding the maintenance burden.   Of course on top of that, getting the info on if  the customers care might be more work than just doing it. :)